core/modules/installer/installer.browser.inc

```diff
@@ -252,8 +252,8 @@ function installer_browser_fetch_results($filters) {
     $results = backdrop_json_decode($response->data);
   }
   else {
-    backdrop_set_message(t("Encountered an error when trying to fetch results from @name. Error @code : @message",
-      array('@name' => $server['name'], '@code' => $response->code, '@message' => $response->error)));
+    backdrop_set_message(t('Encountered an error when trying to fetch projects from %installer_server. Error @code : @message',
+      array('%installer_server' => check_plain($server['name']), '@code' => $response->code, '@message' => $response->error)), 'error');
```

Comment:
This is very hard to read, I'd suggest to display array items on their individual lines:

```
 backdrop_set_message(t('Encountered an error when trying to fetch projects from %installer_server. Error @code : @message', array(
   '%installer_server' => check_plain($server['name']),
   '@code' => $response->code,
   '@message' => $response->error,
 )), 'error');
```